pkg/metadata/gohai/gohai.go

```diff
@@ -65,3 +74,12 @@ func getGohaiInfo() *gohai {
 
 	return res
 }
+
+var docker0Detector = sync.Once{}
+
+func detectDocker0() {
+	docker0Detector.Do(func() {
+		iface, _ := net.InterfaceByName("docker0")
+		docker0Detected = iface != nil
+	})
+}
```

Comment:
I put the var down here and not at the top of the file, because it's used exacty once in the file below.